api.txt

```diff
@@ -0,0 +1,56 @@
+* `wrench.register_node(node.name, definition)`
+
+definition:
+
+    {
+        lists = list
+        -- inventory list
+
+        metas = table
+        -- Table for node metadata:
+            -- name: Meta name
+            -- type: Data type:
+                -- `wrench.META_TYPE_FLOAT` for float
+                -- `wrench.META_TYPE_STRING` for string
+                -- `wrench.META_TYPE_INT` for integer
+
+        description = nil,
+        -- Default behaver:
+            -- when `lists` is not `nil`
+                -- `minetest.registered_nodes[node.name].description` + " with items"
+            -- when `lists` is `nil`
+                -- `minetest.registered_nodes[node.name].description` + " with configuration"
+        description = `wrench.description_with_items`,
+        -- Description of the node + " with items"
+        description = `wrench.description_with_configuration`,
+        -- Description of the node + " with configuration"
+        description = function(pos, meta, node, player),
+        -- Function returns description
+
+        drop = nil,
+        -- Default dropped item is the node itself
+        drop = true,
+        -- Use `minetest.registered_nodes[node.name].drop`
+        drop = "default:cobble",
+        -- Name of dropped item
+
+        owned = nil,
+        -- Default: anyone can pickup this node
+        owned = true,
+        -- Only `metas["owner"]` can pickup this node
+
+        timer = nil,
+        -- Default
+        timer = true,
+        -- Save NodeTimers
+
+        before_remove = nil,
+        -- Default
+        before_remove = function(pos, meta, node, player)
+        -- Function called before 'minetest.remove_node(pos)'
+
+        after_place = nil,
+        -- Default
+        after_place = function(pos, player, itemstack, pointed_thing),
+        -- Function called after place node
+    }
```

Comment:
Maybe make it md instead of plain txt from beginning? For better formatting: API.md